client/store/lineItems.js

```diff
@@ -0,0 +1,76 @@
+import axios from "axios";
+
+/**
+ * ACTION TYPES
+ */
+
+const DELETE_BEER = "DELETE_BEER";
+const LOAD_CART = "LOAD_CART";
+const ADD_BEER = "ADD_BEER";
+const ADD_WINE = "ADD_WINE";
+const UPDATE_LINEITEM = "UPDATE_LINEITEM";
+
+/**
+ * ACTION CREATORS
+ */
+
+const _addBeer = (beer) => ({ type: ADD_BEER, beer });
+
+const _addWine = (wine) => ({ type: ADD_WINE, wine });
```

Comment:
these two can be combined
```suggestion
const _addProduct = (product) => ({ type: ADD_PRODUCT, product });
```